src/Router.js

```diff
@@ -307,7 +306,7 @@ var Router = React.createClass({
   },
 
   _renderScene(route, navigator) {
-    return cloneElement(route.component, route.props);
+    return createElement(route.component, route.props);
```

Comment:
Maybe do a check for if route.component is undefined and use createElement else use clone